src/convertToObject.js

```diff
@@ -1,12 +1,21 @@
-'use strict';
-
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styleString) {
+  const styleObject = {};
+  const lines = styleString.split(';');
```

Comment:
Consider using a more descriptive variable name than 'lines' to improve code readability. For example, 'styleDeclarations' would convey the purpose of the variable more clearly.